src/drive/web/modules/move/MoveModal.jsx

```diff
@@ -201,13 +207,17 @@ export class MoveModal extends React.Component {
             )
           }}
         </Query>
-        <Footer
-          onConfirm={this.moveEntries}
-          onClose={onClose}
-          targets={entries}
-          currentDirId={folderId}
-          isMoving={isMoveInProgress}
-        />
+        <RefreshableSharings>
+          {({ refresh }) => (
+            <Footer
+              onConfirm={() => this.moveEntries(refresh)}
```

Comment:
Have you considered something like `onConfirm={async () => { await this.moveEntries(); refresh()}}`? That way `moveEntries` doesn't have to know about a callback.

Reply:
I think your proposal is so much better. Thanks!

Reply:
In fact we can't since we need also to call the callback after a cancel...